src/dom/dom.js

```diff
@@ -505,30 +819,67 @@ p5.prototype.createSlider = function (min, max, value, step) {
 };
 
 /**
- * Creates a `&lt;button&gt;&lt;/button&gt;` element in the DOM.
- * Use .size() to set the display size of the button.
- * Use .mousePressed() to specify behavior on press.
+ * Creates a `&lt;button&gt;&lt;/button&gt;` element.
+ *
+ * The first parameter, `label`, is a string that sets the label displayed on
+ * the button.
+ *
+ * The second parameter, `value`, is optional. It's a string that sets the
+ * button's value. See <a href="#/p5.Element/value">p5.Element.value()</a> for
```

Comment:
Think it's worth linking to MDN here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#value for an explanation of why a button might need a value and how it gets used? It's definitely an edge case, I didn't even realize this was a thing until a few minutes ago, but I just had to go google it to see what it might mean 😅